### PR TITLE
fix: improve namespace and variables naming

### PR DESCRIPTION
**Pull request by MaferMazu**
## Description

This PR:
- Change the namespace `App` to `OpenedXCommerce.`
- Change the attribute and variable names to add `openedx` as the prefix, e.g., `enrollment_course_id` to `openedx_enrollment_course_id.`
- Add prefixes to some style variables.

## Testing instructions

For testing purposes, you can:
1. Enter in the WordPress admin of this stage https://share.1password.com/s#mjFE_KtE55oS9OrVUCMw0aZw8r4BvF-6ONKeFxJ2BuI
2. Test the API calls in Settings > Open edX Sync Plugin (Saving and generating JWT tokens).
3. Create enrollment requests or save them in WordPress's Open edX Sync option.

Note:
- I tested locally and remotely in a new or existing environment.

## Additional information

This PR is for avoiding WordPress's common mistakes, in particular, the following:

> 4) Missing or inadequate function prefixes
> (issue found in approx. 60% of first reviews)
> 
> All active plugins in a WordPress installation will be loaded during execution. When two elements use the same name, a collision will happen. With more than 80k plugins in the repository, collisions are increasingly common.
> 
> That's why we need plugins to prefix their: function names, class names, namespaces, defines, option names, nonces and own input names with something that is related to this plugin, sufficiently complex and at least 5 characters long.
> 
> Example: a plugin named "Moon forms" could choose "moofor_ " or "moonforms_ " as a prefix but "mf_ ", "moon_ " or "forms_ " would be bad choices as they are too short, too generic or both.
> 
> This rule applies not only to function and class names (where namespaces are the best alternative) but also to defines, saving options, nonces and own input names, for example:
> 
> function moonforms_submit_button(){
> class Moonforms_Form {
> namespace Moonforms\Submit;
> 
> define('MOONFORMS_PLUGIN_URL', plugin_dir_url(__FILE__) );
> define('MOONFORMS_PLUGIN_DIR', plugin_dir_path(__FILE__) );
> 
> update_option( 'moonforms-default-email', $email );
> 
> wp_nonce_field( 'moonforms-form', 'moonforms-nonce' );
> if ( !empty( $_POST['moonforms-nonce'] ) && wp_verify_nonce( sanitize_text_field( wp_unslash ( $_POST['moonforms-nonce'] ) ), 'moonforms-form' ) ) {
> 
> <input name="moonforms-email" type="email">
> $email = sanitize_email( $_POST['moonforms-email'] );

**Note**
Before merging this PR, I want to ensure we are not editing some general CSS styles and colliding with other plugins. 

## Checklist for Merge

- [x] Tested in a remote environment
- [x] Updated documentation
- [x] Rebased master/main

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @MaferMazu! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by MaferMazu** [comment]
I would like to add a few changes to the CSS variable naming (I want to ensure we are not editing some general CSS styles and colliding with other plugins), but if you have early feedback, please let me know.

**Review by felipemontoya — COMMENTED** [review]
@MaferMazu I looked at the changes and it looks reasonable.

In the review it looks like you didn't change the names of anything that becomes a post_metadata, right?
If I'm correct then we should be able to merge and the changes would be backwards compatible with enrollment_requests that were already sent with the old var names. We can verify this by uploading this code in the staging and testing the admin page for the old requests. If everything works we could go ahead and merge.

File: admin/class-openedx-commerce-admin.php [hunk 1]
```diff
@@ -470,11 +470,11 @@ public function items_enrollment_request( $courses, $order_id, $billing_email, $
...
 			$enrollment_arr = array(
-				'enrollment_course_id'    => $course_id,
-				'enrollment_email'        => $billing_email,
-				'enrollment_mode'         => $course_mode,
-				'enrollment_request_type' => $request_type,
-				'enrollment_order_id'     => $order_id,
+				'openedx_enrollment_course_id' => $course_id,
+				'openedx_enrollment_email'     => $billing_email,
+				'openedx_enrollment_mode'      => $course_mode,
+				'enrollment_request_type'      => $request_type,
```
Comment (feanil) [inline admin/class-openedx-commerce-admin.php:476]: Should these other two variables also change for consistency?

**Review by feanil — COMMENTED** [review]
Generally looks good to me but I had one question about a few variables that were not renamed.

File: admin/class-openedx-commerce-admin.php (hunk 1 above)
Comment (MaferMazu) [reply admin/class-openedx-commerce-admin.php]: Yes, changed!! Thanks.

**Comment by MaferMazu** [comment]
Yes, @felipemontoya, I am not touching the post_metadata variables. I have tested the plugin locally and in the Stage with the changes. You can double-check in the Stage with these credentials: https://share.1password.com/s#mjFE_KtE55oS9OrVUCMw0aZw8r4BvF-6ONKeFxJ2BuI.

And I renamed the variables for consistency, as @feanil suggested.

About the CSS variables I wanted to check, I checked, and they don't change the behavior of other plugins.

I think we can go with this and send the ZIP file.

**Review by felipemontoya — APPROVED** [review]
Thanks for the updates and fixes @MaferMazu.

I think we can merge this and present the zip for publication

**Comment by openedx-webhooks** [comment]
@MaferMazu 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/openedx-wordpress-ecommerce&entry.867055334=https://github.com/openedx/openedx-wordpress-ecommerce/pull/65&entry.1484655318=https://github.com/MaferMazu&entry.752974735=2023-12-15+02:00&entry.1917517419=2023-12-20+21:12&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**